lib/modules/versioning/oci-helm/index.ts

```diff
@@ -0,0 +1,109 @@
+import semver from 'semver';
+import { regEx } from '../../../util/regex';
+import { GenericVersion, GenericVersioningApi } from '../generic';
+import type { NewValueConfig, VersioningApi } from '../types';
+
+export const id = 'oci-helm';
+export const displayName = 'OCI Helm';
+export const urls = [
+  'https://semver.org/',
+  'https://helm.sh/docs/chart_best_practices/dependencies/#versions',
+  'https://github.com/Masterminds/semver#basic-comparisons',
+  'https://helm.sh/docs/topics/registries/#oci-feature-deprecation-and-behavior-changes-with-v380',
+];
+export const supportsRanges = false;
+
+// Combining...
+// - https://github.com/Masterminds/semver/blob/2f39fdc11c33c38e8b8b15b1f04334ba84e751f2/version.go#L42
+// - https://regex101.com/r/vkijKf/1/ (official one from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string)
+// ...into https://regex101.com/r/Visopn/1 with group names
+// ...prepended by range group to properly parse range values
+const versionRegex = regEx(
+  /^(?<range>~|<|<=|>|>=|\^)?v?(?<major>[0-9]\d*)(\.(?<minor>[0-9]\d*))?(\.(?<patch>[0-9]\d*))?(?:-(?<prerelease>(?:[0-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+(?<metadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/,
+);
+
+class OciHelmVersioningApi extends GenericVersioningApi {
+  private _oci2semver(value: string): string {
```

Comment:
```suggestion
  private _oci2semver(value: string | null| undefined): string {
```
why do we need the conditional below 👇? do we accept null or undefined?